src/cmake-tools.ts

```diff
@@ -982,7 +982,11 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
         return false;
     }
 
+    private isKitChanged: boolean = false;
```

Comment:
This variable only serves the purpose of letting us know that a new terminal needs to be created so I think the name should encode that purpose instead of encoding what happened.  If the user never launches a terminal, then this will stay true forever and there's no obvious time when it should be changed to false.

Consider something along the lines of `launchTerminalStale` or `refreshLaunchEnvironment` instead.